scipy/linalg/tests/test_basic.py

```diff
@@ -734,6 +734,7 @@ def test_simple_complex2(self):
                                                 [-4-5.j, -3+4.j],
                                                 [6.j, 2-3.j]]))
 
+    @pytest.mark.xfail(reason="See gh-22265")
```

Comment:
Might be worth fixing this in this PR. Up to you.

Reply:
If you have a simple solution for it, perhaps. I meant to sign up to fix gh-22263, but not every pre-existing bug that I exposed while fixing it : )

Reply:
Now fixed here.